acceptance-tests/src/test/java/tech/pegasys/signers/keystorage/hashicorp/tests/HashicorpVaultAccessAcceptanceTest.java

```diff
@@ -73,13 +76,17 @@ void cleanup() {
   void keyCanBeExtractedFromVault() throws IOException {
     hashicorpNode = HashicorpNode.createAndStartHashicorp(docker, false);
 
+    final String hashicorpSecretHttpPath =
+        hashicorpNode.addSecretsToVault(
+            Collections.singletonMap(SECRET_KEY, SECRET_VALUE), "acceptanceTestSecret");
+
```

Comment:
nit: We should use the utlity method directly here to construct the URL. The intent of "addSecretsToVault" should either be successful or failed.